pabot/pabot.py

```diff
@@ -283,9 +290,20 @@ def _writer():
         message = MESSAGE_QUEUE.get()
         print message
 
-def _write(message):
+def _write(message, color=None):
+    if _is_output_coloring_supported() and color:
+        message = _wrap_with(color, message)
     MESSAGE_QUEUE.put(message)
 
+def _wrap_with(color, message):
+    return "%s%s%s" % (color, message, Color.ENDC)
+
+def _is_output_coloring_supported():
+    if any([not sys.stdout.isatty(),
+            not os.name in Color.SUPPORTED_OSES]):
+        return False
+    return True
+
```

Comment:
I wouldn't use 
  
    if condition: 
          return not condition
    return not condition

Instead just:

    def _is_output_coloring_supported():
           return sys.stdout.isatty() and os.name in Color.SUPPORTED_OSES